doc/api/process.md

```diff
@@ -308,7 +308,8 @@ added:
 * `err` {Error} The uncaught exception.
 * `origin` {string} Indicates if the exception originates from an unhandled
   rejection or from synchronous errors. Can either be `'uncaughtException'` or
-  `'unhandledRejection'`.
+  `'unhandledRejection'`. The latter is only used in conjunction with the
+[`--unhandled-rejections`][] flag set to `strict` and an unhandled rejection.
```

Comment:
```suggestion
[`--unhandled-rejections`][] flag set to `strict` or `throw` and an unhandled
rejection.
```

Reply:
done.  
I added it in the doc for `uncaughtException` too, as both were impacted by the change